src/main/ruby/core/truffle/kernel_operations.rb

```diff
@@ -40,3 +40,25 @@ def self.define_read_only_global(name, getter)
 alias $: $LOAD_PATH
 alias $-I $LOAD_PATH
 alias $" $LOADED_FEATURES
+
+Truffle::KernelOperations.define_hooked_variable(
+  :$/,
+  -> { Truffle::KernelOperations.global_variable_get(:$/) },
+  -> v { if v && !Truffle::Type.object_kind_of?(v, String)
+           raise TypeError, '$/ must be a String'
+         end
+         Truffle::KernelOperations.global_variable_set(:$/, v) })
+
+$/ = "\u000a".freeze
+
+alias $-0 $/
+
+Truffle::KernelOperations.define_hooked_variable(
+  :$,,
+  -> { Truffle::KernelOperations.global_variable_get(:$,) },
+  -> v { if v && !Truffle::Type.object_kind_of?(v, String)
+           raise TypeError, '$/ must be a String'
```

Comment:
`$,`

Why is `$,` added in this commit?
=> it's in the next commit.
Maybe merge those pairs of commits which mixes changes of 2 variables?